app/bundles/CampaignBundle/Entity/ReplicaConnectionTrait.php

```diff
@@ -19,16 +19,16 @@ trait SlaveConnectionTrait
      *
      * @return Connection
      */
-    private function getSlaveConnection(ContactLimiter $limiter = null)
+    private function getReplicaConnectionTrait(ContactLimiter $limiter = null)
```

Comment:
I think that the "Trait" should not be in the method name. Also, it'd be nice to have the return type.
```suggestion
    private function getReplicaConnection(ContactLimiter $limiter = null): Connection
```